hazelcast/src/main/java/com/hazelcast/jet/accumulator/AccumulatorSerializerHooks.java

```diff
@@ -276,12 +276,11 @@ public int getTypeId() {
                 @Override
                 public void write(@Nonnull ObjectDataOutput out, @Nonnull PickAnyAccumulator acc) throws IOException {
                     out.writeObject(acc.get());
-                    out.writeLong(acc.count());
```

Comment:
Can we somehow send this accumulator to the older versions? Let's say the server has version 5.0, but the client is 5.2. Do we expect this to work?

Reply:
These are the compatibility guarantees we provide for Jet.

https://jet-start.sh/docs/operations/version-compatibility#summary-table

Member version must have a matching PATCH version to execute jobs, but I think that currently there is no check for this and it just blows up.

For client-member they must have a matching MINOR version, so this PR is ok in 5.2. I also think this is ok to backport, because the accumulator is never sent between client and member (only a Supplier is).

Reply:
And it's also not sent to the client.